backend/audit/test_validators.py

```diff
@@ -33,6 +40,229 @@ def test_validation_is_applied(self):
         simple = SchemaValidityTest.SIMPLE_CASE
         validate_federal_award_json(simple)
 
+    def test_prefix_under_ten(self):
+        """
+        Prefixes between 00 and 09 should fail
+        """
+        simple = jsoncopy(SchemaValidityTest.SIMPLE_CASE)
+
+        # pick a prefix between 00 and 09 (invalid)
+        prefix = f"{randrange(10):02}"
+        # pick an extension beteween 100 and 999 (valid)
+        extension = f"{randrange(100, 1000):03}"
+
+        simple["FederalAwards"]["federal_awards"][0][
+            "program_number"
+        ] = f"{prefix}.{extension}"
+
+        self.assertRaises(ValidationError, validate_federal_award_json, simple)
+
+    def test_prefix_over_99(self):
+        """
+        Prefixes over 99 should fail
+        """
+        simple = jsoncopy(SchemaValidityTest.SIMPLE_CASE)
+
+        # pick a prefix between 100 and 999 (invalid)
+        prefix = f"{randrange(100, 1000):03}"
+        # pick an extension beteween 100 and 999 (valid)
+        extension = f"{randrange(100, 1000):03}"
```

Comment:
Since we discovered that `001` was a valid extension, I think this should be `extension = f"{randrange(1,1000):03}"`

Reply:
fixed - thanks!